os/compression/compress_read.c

```diff
@@ -402,7 +608,12 @@ int compress_init(int filfd, uint16_t offset, off_t *filelen)
  ****************************************************************************/
 void compress_uninit(void)
 {
-	/* Freeing memory allocated to read_buffer and out_buffer for file decompression */
-	free(buffers.read_buffer);
-	free(buffers.out_buffer);
+	/* Freeing memory allocated to read_buffer and blockcaches for file decompression */
+	free(read_buffer);
```

Comment:
1. kmm_free
2. Could you let me know where definition is for `read_buffer`?
3. Could you specify the usages of this buffer in a name? I mean like `caching_buffer` or `decomp_buffer` or `compressed_data_buffer` or something else/

Reply:
1. Changed to kmm_free
2. Definition is in this file only (Line no. 58)
3. Changed name to compressed_data_buffer